src/test/resources/alma/HT012734833.json

```diff
@@ -25,6 +25,78 @@
       "label" : "lobid Organisation"
     },
     "id" : "https://lobid.org/item/991000300959706445"
+  }, {
+    "id" : "https://lobid.org/item/5333897520006444",
+    "type" : [ "Item" ],
+    "heldBy" : {
+      "id" : "http://lobid.org/organisations/DE-632#!",
+      "label" : "lobid Organisation",
+      "electronicLocator" : "https://eu04.alma.exlibrisgroup.com/view/uresolver/49HBZ_NETWORK/openurl?u.ignore_date_coverage=true&portfolio_pid=5333897520006444&Force_direct=true",
```

Comment:
I think this is the wrong level for this key. It should cling directly to Item, see https://id.loc.gov/ontologies/bibframe.html#c_Item.
Same for "sublocation".

Reply:
resolved with https://github.com/hbz/lobid-resources/pull/1121/commits/f6e40fa5ff26c5fc14bb4d31a39c41ff79803399.